src/LinuxTracing/SwitchesStatesNamesVisitor.cpp

```diff
@@ -112,11 +113,12 @@ void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
                            event_data.was_created_by_pid);
 }
 
-void SwitchesStatesNamesVisitor::Visit(uint64_t event_timestamp,
-                                       const SchedSwitchPerfEventData& event_data) {
-  // Note that context switches with tid 0 are associated with idle CPU, so we never consider them.
-
+template <typename SchedSwitchEventData>
+void SwitchesStatesNamesVisitor::VisitSchedSwitch(uint64_t event_timestamp,
+                                                  const SchedSwitchEventData& event_data,
+                                                  bool wait_for_callstack) {
```

Comment:
Didn't we decide to call this `has_switch_out_callstack`?

Reply:
Done.